src/pp_roaming_protocol.erl

```diff
@@ -198,13 +204,14 @@ handle_prstart_ans(#{
     } = DLMeta
 }) ->
     {ok, TransactionID, Region, PacketTime, _, _} = parse_uplink_token(Token),
-
+    Plan = lora_plan:region_to_plan(Region),
+    DataRate = lora_plan:datarate_to_string(Plan, DR),
     DownlinkPacket = blockchain_helium_packet_v1:new_downlink(
         pp_utils:hexstring_to_binary(Payload),
         _SignalStrength = 27,
         pp_utils:uint32(PacketTime + ?JOIN1_DELAY),
         Frequency,
-        pp_lorawan:index_to_datarate(Region, DR),
```

Comment:
Is `pp_lorawan:index_to_datarate/2` used anywhere? If not, should we remove it?

Reply:
That was the idea :-)  Done.